documentation/modules/overview/con-key-features-kafka-connect.adoc

```diff
@@ -0,0 +1,141 @@
+// This module is included in:
+//
+// overview/assembly-key-features.adoc
+
+[id="key-features-kafka-connect_{context}"]
+= How Kafka Connect streams data
+
+[role="_abstract"]
+Kafka Connect uses connector instances to integrate with other systems to stream data.
+
+Kafka Connect loads existing connector instances on start up and distributes data streaming tasks and connector configuration across worker pods.
+Workers run the tasks for the connector instances.
+Each worker runs as a separate pod to make the Kafka Connect cluster more fault tolerant.
+Work is distributed evenly through load balancing and is divided into tasks.
```

Comment:
I think we can drop this sentence. Or at least rework it. Firstly, there's nothing in connect which really tries to balance _load_ in a meaningful way. Tasks are distributed evenly, but there's nothing preventing, or adjusting things, if the hottest tasks land on the same worker. Secondly, I think we've made the "...is divided into tasks" point several time already. Even in this paragraph we've mentioned "task" twice already.